src/Core/Settings/GlobalSettings.cs

```diff
@@ -483,7 +483,7 @@ public class StripeSettings
 
         public class TwoFactorAuthSettings : ITwoFactorAuthSettings
         {
-            public bool EmailOnNewDeviceLogin { get; set; } = true;
+            public bool EmailOnNewDeviceLogin { get; set; }
```

Comment:
I know `bool` is default false, but be explicit here since we want some confirmation
```suggestion
            public bool EmailOnNewDeviceLogin { get; set; } = false
```